docs/builds/guides/frameworks/vuejs.md

```diff
@@ -0,0 +1,645 @@
+
+```js
+import Vue from 'vue';
+import CKEditor from '@ckeditor/ckeditor5-vue';
+import ClassicEditor from '@ckeditor/ckeditor5-build-classic';
+
+Vue.use( CKEditor, {
+	editors: {
+		classic: ClassicEditor
+	}
+} );
+```
+
+<info-box>
+	Check out the [plugin configuration](#plugin-configuration) section to learn more.
+</info-box>
+
+<info-box>
+	You can always configure the component locally if you do not want to [specify editor builds](#local-build-configuration) at that point or just to [avoid using `Vue.use()`](#local-component-registration).
+</info-box>
+
+The following example showcases a single–file component of the application. Use the `<ckeditor>` component in your template:
+
+* The [`editor`](#editor) directive specifies the editor build.
+* The [`v-model`](#v-model) directive enables an out–of–the–box two–way data binding.
+* The [`config`](#config) directive helps you pass the configuration to the editor instance.
+
+```html
+<template>
+	<div id="app">
+		<ckeditor editor="classic" v-model="editorData" :config="editorConfig"></ckeditor>
+	</div>
+</template>
+
+<script>
+	export default {
+		name: 'app',
+		data() {
+			return {
+				editorData: '<p>Content of the editor.</p>',
+				editorConfig: {
+					// Configuration of the editor.
+				}
+			};
+		}
+	}
+</script>
+```
+
+<info-box>
+	See the list of supported [directives](#component-directives) and [events](#component-events) that will help you configure the component.
+</info-box>
+
+### Using the component locally
+
+#### Local build configuration
+
+If a per–view editor build configuration is what suits you best, you can skip the [`editors`](#editors) option in `Vue.use()`:
+
+```js
+Vue.use( CKEditor );
+```
+
+and then pass the build of your choice in the [`editor`](#editor) directive:
+
+```html
+<template>
+	<div id="app">
+		<ckeditor :editor="editorType" ... ></ckeditor>
+	</div>
+</template>
+
+<script>
+	export default {
+		name: 'app',
+		data() {
+			return {
+				editorType: ClassicEditor,
+
+				// ...
+			};
+		}
+	}
+</script>
+```
+
+#### Local component registration
+
+If you do not want the CKEditor component to be enabled globally, you can entirely skip the `Vue.use( CKEditor, { ... } )` part. Instead, configure it in the `components` property of your view:
+
+```html
+<template>
+	<div id="app">
+		<ckeditor :editor="editorType" ... ></ckeditor>
+	</div>
+</template>
+
+<script>
+	export default {
+		name: 'app',
+		components: {
+			// Use the CKEditor component in this view.
+			ckeditor: CKEditor.component
+		},
+		data() {
+			return {
+				editorType: ClassicEditor,
+
+				// ...
+			};
+		}
+	}
+</script>
+```
+
+## Using CKEditor built from source
```

Comment:
There are two main ways to include the editor – via a build and from source. In the latter scenario, we should not use the term "editor build" because it does not apply. There's no intermediate file create for that.

Therefore, the "Advanced setup" guide, even though it describes two methods, it does not use the term "editor build" in `builds/guides/integration/advanced-setup.html#scenario-2-building-from-source`. 

I think that the Vue's guide should follow the React guide where in the "from source" section we use the most direct, close method possible – importing everything directly. No additional file is created. The goal there is to show the absolute minimum setup, something most "native". Then we can link to the "Advanced setup" guide with "you can learn about other methods and scenarios in".

Reply:
I rephrased the documentation. Check out and tell me if it makes sense now.